src/test/java/com/google/api/codegen/protoannotations/testdata/java_library_no_gapic_config.baseline

```diff
@@ -318,6 +341,31 @@ public class LibraryServiceClient implements BackgroundResource {
     return operationsClient;
   }
 
+  // AUTO-GENERATED DOCUMENTATION AND METHOD
+  /**
+   * Creates a shelf, and returns the new Shelf.
+   * RPC method comment may include special characters: &lt;&gt;&amp;"`'{@literal @}.
+   *
+   * Sample code:
+   * <pre><code>
+   * try (LibraryServiceClient libraryServiceClient = LibraryServiceClient.create()) {
+   *   Shelf shelf = Shelf.newBuilder().build();
+   *   Shelf response = libraryServiceClient.createShelf(shelf);
+   * }
+   * </code></pre>
+   *
+   * @param shelf The shelf to create.
+   * @throws com.google.api.gax.rpc.ApiException if the remote call fails
+   */
+  public final Shelf createShelf(Shelf shelf) {
```

Comment:
This generated method, along with other flattened methods that use a STATIC_TYPES param, also exists in the GAPIC-config-generated java_library.baseline.